esp/templates/survey/answers.html

```diff
@@ -10,7 +10,7 @@
 {% load survey %}
 {% load main %}
 
-<table width="100%">
+<table width="600"> 
```

Comment:
We've been trying to move to percentages for widths lately. It's more accessible.